sdk/datafactory/arm-datafactory/src/models/index.ts

```diff
@@ -13748,14 +13762,15 @@ export interface ActivityPolicy {
 export interface ExecuteDataFlowActivityTypePropertiesCompute {
   /**
    * Compute type of the cluster which will execute data flow job. Possible values include:
-   * 'General', 'MemoryOptimized', 'ComputeOptimized'
+   * 'General', 'MemoryOptimized', 'ComputeOptimized'. Type: string (or Expression with resultType
+   * string)
    */
-  computeType?: DataFlowComputeType;
+  computeType?: any;
```

Comment:
This and other changes around this line where the type is being changed to `any` is suspicious. Do we know what triggered this?

Reply:
This is because of https://github.com/Azure/azure-rest-api-specs/pull/10897/files. In datafactory, they allow user to input either an enum or an expression, but currently we are using swagger 2.0 which doesn't allow the "oneOf" that the schema can be multiple possible schema and "oneOf" can only be supported in swagger 3.0. Therefore, service team use type: object to bypass it.

Reply:
@joheredi Does this sound ok to you?

Reply:
Yeah, I think that is reasonable since OpenApi doesn't seem to allow oneOf. The generated output seems to be what I expect for an object type with no properties defined